js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -1,3 +1,18 @@
-function alternatingCharacters(arr) {}
+function getMinimumDeletion(string) {
+  let previousChar = "";
```

Comment:
please don't use short word or abbreviation for variable

Reply:
Sure, will take care of these mistakes. Also will update this code with `previousCharacter`.

Reply:
Worked on the review and updated the PR with this commit 13e2d73.